src/scripts/models/content.coffee

```diff
@@ -266,3 +266,13 @@ define (require) ->
         $('#publish-modal').modal()
       else
         $('#reject-publish-modal').modal()
+
+    license: () ->
+      license = undefined
+      @setLicense = (value) ->
+        license = value
+        return
+
+      @getLicense = ->
+        license
+      return
```

Comment:
I don't think you want to be returning nothing here in a 'get' method.  Since CoffeeScript implicitly returns the value of the last line, you can just write this as `@getLicense = -> license`.

Although should the `license` value actually be stored on the content model itself, rather than just existing in the license method closure?